R/upgrade.R

```diff
@@ -46,6 +46,8 @@ dm_upgrade <- function(dm, quiet) {
     def <- unclass(dm)$def
     def$pks <- map(def$pks, mutate, autoincrement = FALSE) %>%
       vctrs::as_list_of(new_pk())
+    def$uks <- list_of(new_uk())
```

Comment:
Are we importing unique keys implicitly created by foreign keys here?

Reply:
now yes, see 75d5ad42b281db4c4303c6f4e9a0efbcca2a1897